orbmain/src/main/java/com/sun/corba/ee/impl/transport/ConnectionImpl.java

```diff
@@ -225,8 +225,13 @@ private ConnectionImpl(ORB orb,
         this.contactInfo = contactInfo;
 
         try {
+            TcpTimeouts tcpConnectTimeouts = orb.getORBData().getTransportTcpConnectTimeouts();
             defineSocket(useSelectThreadToWait,
-                    orb.getORBData().getSocketFactory().createSocket(socketType, new InetSocketAddress(hostname, port)));
+                    orb.getORBData().getSocketFactory().createSocket(
+                            orb.getORBData().connectionSocketType(),
```

Comment:
Yet one question - why replaced the `socketType` parameter with `orb.getORBData().connectionSocketType()`? It seems it did not change anything in test results, but ...